azext_iot/_params.py

```diff
@@ -879,6 +892,18 @@ def load_arguments(self, _):
             options_list=["--auth-type"],
             arg_type=hub_auth_type_dataplane_param_type,
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries", "--cmq"],
+            help="An alternative way of input style(narg key-value pair) to --metrics and intended to replace it in the future.",
```

Comment:
If there is no specificity in the help description to the type of resource then the arg definitions doesn't need to be repeated twice. Note other arg definition descriptions referring to "deployment" here and to "configuration"  for the hub configuration commands, that is the reason for the duplicates.

So if you don't think language specificity is value add, then we should remove the duplicates and define the new args  once at a higher scope (iot).

Reply:
definition moved to iot scope